### ISSUE TITLE
Anonymous User Submission

### ISSUE THREAD

**Issue by Bronson-Brown-deVost** [Scripta-Qumranica-Electronica/ScrollEditor#43]
Submitter: Jonathan
I had trouble with the "adjust image" function. I tried 4Q325 frg 1 and 4Q554 frg 3. In both of them I could only see the color image, not the IR neither the RL. Theese three did appear in the list but when I turned them on, they did not appear. Note also that in 4Q554 3 I had only 3 in the list (did not have RL right). I use Windows 10 and Firefox.

**Comment by Bronson-Brown-deVost** [comment]
@Bronson-Brown-deVost , please confirm the problem and check the database to see why these particular images might be having problems.

**Comment by tsviar** [comment]
Indeed, 4Q554 3, has only 3 image adjustment options, but all options work, in all the artefacts mentioned above.
Branch development and fix/GIT-hi-priority-fixes.
![image](https://user-images.githubusercontent.com/16132723/116585114-b70c0d00-a920-11eb-97b4-7124ca4078f7.png)
![image](https://user-images.githubusercontent.com/16132723/116585275-e3278e00-a920-11eb-8dca-43f4aaf84f37.png)

**Comment by Bronson-Brown-deVost** [comment]
Yeah, I see all the images working properly now. Only 4Q554 frg. 3 is weird because the image with left raking light seems to be missing. @OritandJoe, can you check on this?  It is p319/1 f2.

**Comment by oritRosengarten** [comment]
Yes, it is missing in our image server as well, we will take care of it (there are a few more missing), it will take a while
